package.json

```diff
@@ -8,42 +8,47 @@
   },
   "scripts": {
     "lint:tests": "eslint --ext .js --fix tests/",
+    "ci": "npx codeceptjs run-multiple parallel --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@dbaas)^(?!.*@ami-upgrade)^(?!.*@pmm-upgrade)^(?!.*@qan)^(?!.*@nightly)'",
```

Comment:
in jenkins pipeline it is [hardcoded](https://github.com/Percona-Lab/jenkins-pipelines/blob/cd0b958b9e505aaa204c8f0a4bf4b03c89d8fe3f/pmm/pmm2-ui-tests.groovy#L282). the idea is to use run `npm run ci` in pipelines instead.

Reply:
thanks